ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/operations/SlashableAttestation.java

```diff
@@ -122,4 +126,17 @@ public Bytes getCustody_bitfield() {
   public void setCustody_bitfield(Bytes custody_bitfield) {
     this.custody_bitfield = custody_bitfield;
   }
+
+  public Bytes32 hash_tree_root() {
+    return HashTreeUtil.merkleize(
+        Arrays.asList(
+            HashTreeUtil.hash_tree_root(SSZTypes.LIST_OF_BASIC,
+                validator_indices.stream()
+                    .map(item -> SSZ.encodeUInt64(item.longValue()))
+                    .collect(Collectors.toList())),
+            data.hash_tree_root(),
+            // TODO Fix this.
```

Comment:
Just a reminder, after you're done with container hash tree root, we gotta add attestationData.hash_tree_root here

Reply:
or actually maybe we can just add that now?